Documentation/contributing/development/contributing_guide.rst

```diff
@@ -183,13 +211,64 @@ Getting a pull request merged
 
 #. Owners of the repository will automatically adjust the labels on the pull
    request to track its state and progress towards merging.
+
 #. Once the PR has been reviewed and the CI tests have passed, the PR will be
    merged by one of the repository owners. In case this does not happen, ping
    us on Slack.
 
+#. If reviewers have requested for changes and once those changes are done,
```

Comment:
This one sounds weird, not 100% sure. I don't think you can use `for` after the verb `to request` (even though it is fine after a noun).
```suggestion
#. If reviewers have requested changes and once those changes are done,
```

Reply:
I think "have requested for" is correct, some English native speaker would know better. cc @joestringer

Reply:
"for" is not needed.